linera-service/tests/linera_net_tests.rs

```diff
@@ -350,33 +352,24 @@ async fn test_evm_end_to_end_counter(config: impl LineraNetConfig) -> Result<()>
 
     sol! {
         struct ConstructorArgs {
-            uint256 initial_value;
+            uint64 initial_value;
         }
-        function increment(uint256 input);
+        function increment(uint64 input);
         function get_value();
     }
 
-    let original_counter_value = U256::from(35);
+    let original_counter_value = 35;
     let instantiation_argument = ConstructorArgs {
         initial_value: original_counter_value,
     };
     let instantiation_argument = instantiation_argument.abi_encode();
 
-    let increment = U256::from(5);
+    let increment = 5;
 
     let chain = client.load_wallet()?.default_chain().unwrap();
-    let module = get_example_counter()?;
-
-    let dir = tempfile::tempdir()?;
-    let path = dir.path();
-    let app_file = "app.json";
-    let app_path = path.join(app_file);
-    {
-        std::fs::write(app_path.clone(), &module)?;
-    }
+    let module = get_evm_example_counter()?;
 
-    let contract = app_path.to_path_buf();
-    let service = app_path.to_path_buf();
+    let (contract, service, _dir) = get_contract_service_paths(module)?;
```

Comment:
Are we publishing and storing two distinct blobs (contract and service) with identical content? We should deduplicate that for EVM contracts somehow. (Not in this PR.)

Reply:
Will look at that.

Reply:
Yes, maybe we need to distinguish, and have a separate `BlobType::EvmBytecode` instead?